drafts/SC1-4-1-link-in-text-style.md

```diff
@@ -1,127 +1,172 @@
+---
+rule_id: SC1-4-1-link-in-text-style
+name: Inline links are distinguishable
+test_mode: automatic
 
-# SC1-4-1-a 
-
-This test belongs to [[1.4.1 Use of Color]].
+criteria:
+- 1.4.1 # Use of Color (Level A)
 
+authors:
+- Wilco Fiers
+---
 
 ## Description
-[[SC1-4-1-a]] checks if `&lt;a&gt;` elements meet [[1.4.1 Use of Color]].
-
-This test checks if links within text are visually evident before they receive focus, in which case the test passes. If not, the test checks if the links have a significant contrast difference with the surrounding text. If this is the case, and if the link becomes visually evident when it receives focus and when it is hovered by a cursor, it also passes.
 
+This rule checks that links that are embedded in a block of text can be distinguished in a way that does not rely exclusively on color. If this is the case, the link can only pass if the color contrast is sufficient, and additional cues are given when the link receives focus or hover.
 
 ## Background
+
 - [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/F73.html)
 - [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G183)
 - [C15: Using CSS to change the presentation of a user interface component when it receives focus](http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/C15:)
 - [Color contrast (WCAG 2.0 definition)](http://www.w3.org/TR/WCAG20/#contrast-ratiodef)
 
-
 ## Assumptions
+
 - This test assumes that the 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of G183 technique, but is not specified in the 1.4.1 success criterion.
-- This tests assumes that one of the following shorthand CSS-properties (or the related expanded properties) is used to make the link visually evident: background, border, color, font, or text-decoration.
+- This tests assumes that one of the following shorthand CSS-properties (or the related expanded properties) is used to make the link visually evident: `background`, `border`, `color`, `font`, or `text-decoration`.
 - This test assumes that any use of border will make links sufficiently distinguishable
```

Comment:
redundant to the last point: Use of a `border`, of 1 or more pixels, not set to none, and not with a color of transparent, is assumed to be distinguishable